File: solutions/0961. N-Repeated Element in Size 2N Array/0961.java
class Solution {
  public int repeatedNTimes(int[] A) {
    for (int i = 0; i + 2 < A.length; ++i)
      if (A[i] == A[i + 1] || A[i] == A[i + 2])
        return A[i];

    return A[A.length - 1];
  }
}

File: solutions/0961. N-Repeated Element in Size 2N Array/0961.py
class Solution:
  def repeatedNTimes(self, nums: List[int]) -> int:
    for i in range(len(nums) - 2):
      if nums[i] == nums[i + 1] or nums[i] == nums[i + 2]:
        return nums[i]
    return nums[-1]
